app/src/main/java/io/appium/uiautomator2/model/internal/CustomUiDevice.java

```diff
@@ -86,6 +89,16 @@ public Instrumentation getInstrumentation() {
         return mInstrumentation;
     }
 
+    public UiAutomation getUiAutomation() {
+        int flags = Configurator.getInstance().getUiAutomationFlags();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N)
+        {
```

Comment:
Please format this block, so the opening bracket is on the same line with `if` operator

Reply:
Thanks for the review, fixed this in the latest commit https://github.com/appium/appium-uiautomator2-server/pull/463/commits/162cf19fa825233529073fa04f1d770d1aaffb05